src/icon-manifest/manifestMerger.ts

```diff
@@ -1,49 +1,39 @@
 import * as _ from 'lodash';
-import {
-  IFileCollection,
-  IFolderCollection,
-  IExtensionCollection,
-  IExtension,
-  FileFormat,
-  IIconSchema,
-  IIconGenerator,
-  IFileDefault,
-  IFolderDefault,
-} from '../models';
+import * as model from '../models';
```

Comment:
In fact, I don't really see the benefit of doing such an import. We reduce the size of the import but we lose readability and gain verbosity in our code.